src/controllers/userController.js

```diff
@@ -140,6 +140,38 @@ export const editUser = async (req, res) => {
   }
 };
 
+export const deleteUser = async (req, res) => {
+  const userId = req.params.id;
+
+  try {
+    const user = await User.findById(userId);
+
+    await user.delete();
```

Comment:
I'm getting the following error when I try to delete a user: _user.delete is not a function_

Maybe you can use the `findByIdAndDelete` method instead of `delete`.

Reply:
fixed